.github/workflows/main.yml

```diff
@@ -445,7 +445,18 @@ jobs:
           path: out/linux/arm64
 
   mac-build:
-    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, macos-14]
+        include:
+          - os: macos-latest
+            OCAML_VERSION: 4.08.1
```

Comment:
Not strictly related to this PR, but I don't see why we should default to an old OCaml version here. Even on Windows I've successfully used 4.14.1, so perhaps this would be a good time to update this.